### PR TITLE
Add Poetry DB under Books

### PR DESCRIPTION
**Pull request by AugustasV**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>
Poetry DB API Enables to get instant data from our vast poetry collection.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +239,7 @@ API | Description | Auth | HTTPS | CORS |
 | [LibGen](https://garbage.world/posts/libgen/) | Library Genesis search engine | No | No | Unknown |
 | [Open Library](https://openlibrary.org/developers/api) | Books, book covers and related data | No | Yes | No |
 | [Penguin Publishing](http://www.penguinrandomhouse.biz/webservices/rest/) | Books, book covers and related data | No | Yes | Yes |
+| [PoetryDB](https://github.com/thundercomb/poetrydb#readme) | enables you to get instant data from our vast poetry collection. | No | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:242]: - first character of description is not capitalized
- description should not end with .

```suggestion
| [PoetryDB](https://github.com/thundercomb/poetrydb#readme) | Enables you to get instant data from our vast poetry collection | No | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @AugustasV ;)

Can you resolve this review, please?

**Review by xan187 — APPROVED** [review]

**Review by xan187 — CHANGES_REQUESTED** [review]
change api to my

**Review by matheusfelipeog — APPROVED** [review]
Now it's ok ;)

Thanks for the update @AugustasV

**Pull request merged**